fix Makefile.in.in rather than running `autopoint` as that requires cvs

http://bugs.gentoo.org/142565
http://bugs.gentoo.org/142997

--- po/Makefile.in.in
+++ po/Makefile.in.in
@@ -27,7 +27,7 @@
 INSTALL = @INSTALL@
 INSTALL_DATA = @INSTALL_DATA@
 MKINSTALLDIRS = @MKINSTALLDIRS@
-mkinstalldirs = @MKINSTALLDIRS@
+mkinstalldirs = $(SHELL) `case "$(MKINSTALLDIRS)" in /*) echo "$(MKINSTALLDIRS)" ;; *) echo "$(top_builddir)/$(MKINSTALLDIRS)" ;; esac`
 
 GMSGFMT = @GMSGFMT@
 MSGFMT = @MSGFMT@
